webui/react/src/utils/job.ts

```diff
@@ -51,3 +52,7 @@ export const jobStateToLabel: {[key in JobState]: string} = {
   [JobState.SCHEDULEDBACKFILLED]: 'ScheduledBackfilled',
   [JobState.QUEUED]: 'Queued',
 };
+
+export const orderedSchedulers = new Set(
+  [ Api.V1SchedulerType.PRIORITY, Api.V1SchedulerType.KUBERNETES ],
+);
```

Comment:
**question:** would you mind adding `utils/job.test.ts` with some tests for these utility functions? I forgot to mention it last time 🙏

Reply:
thanks for the reminder.